### PR TITLE
Updated month view to support markilux.

### PR DESCRIPTION
**Pull request by JotamSleonardo**
Sample Screenshot:

Screenshot 2023-02-24 at 11 35 14 AM

### CODE REVIEW

**Comment by raelsj** [comment]
lints sir

**Comment by raelsj** [comment]
please provide a screenshot of both resource app and absence app that will use this.

**Comment by JotamSleonardo** [comment]
> please provide a screenshot of both resource app and absence app that will use this.

@raelsj Resource app have a different calendar UI.

**Comment by raelsj** [comment]
> > please provide a screenshot of both resource app and absence app that will use this.
> 
> @raelsj Resource app have a different calendar UI.

@JotamSleonardo I know. But how if you need to change something in the resource app therefore you need to update the package and the reference of the resource app will get the updated one. and this update won't support the same thing with resource app then.

File: lib/src/day_container.dart [hunk 1]
```diff
@@ -91,8 +96,8 @@ class _DateWidget extends StatelessWidget {
         child: Text(
           '$date',
           style: dateTextStyle?.copyWith(color: isToday ? Colors.white : Colors.black) ??
-              textTheme.headline4!.copyWith(
-                color: isToday ? Colors.white : Colors.black,
+              textTheme.headlineMedium!.copyWith(
```
Comment (raelsj) [inline lib/src/day_container.dart:99]: mind using ?. instead of !.

File: lib/src/event_calendar.dart [hunk 2]
```diff
@@ -196,7 +205,7 @@ class _EventCalendarState extends State<EventCalendar> with SingleTickerProvider
                             day.dayStringValue,
                             style: day.textStyle ??
                                 widget.weekdaysHeaderTextStyle ??
-                                textTheme.bodyText2!.copyWith(color: Colors.white),
+                                textTheme.bodyMedium!.copyWith(color: Colors.white),
```
Comment (raelsj) [inline lib/src/event_calendar.dart:208]: see comment above

File: lib/src/event_item.dart [hunk 3]
```diff
@@ -19,22 +23,20 @@ class EventItem extends StatelessWidget {
...
         padding: const EdgeInsets.symmetric(
-          vertical: 10,
+          vertical: 3,
           horizontal: 5,
         ),
         child: Text(
           event.title,
-          style: event.textStyle ?? textTheme.bodyText2!.copyWith(fontSize: 14),
+          style:
+              event.textStyle ?? textTheme.bodyMedium!.copyWith(fontSize: 10),
```
Comment (raelsj) [inline lib/src/event_item.dart:39]: see above

File: lib/src/month_view.dart [hunk 4]
```diff
@@ -65,15 +71,35 @@ class _MonthViewState extends State<MonthView> {
...
-        Positioned(
-          left: i * widget.dayWidgetSize.width,
-          top: position * (eventHeight + 20) + dateTextHeight,
-          width: width,
-          child: EventItem(event: event),
-        ),
-      );
-      eventWidgetsInDay.add(SizedBox(height: eventHeight + 15));
+
+      // NOTE: This logic will replace the last event as an excess event widget.
```
Comment (raelsj) [inline lib/src/month_view.dart:75]: remove the term `NOTE` your comment explains it all already.

File: lib/src/more_events.dart [hunk 5]
```diff
@@ -6,69 +6,33 @@ class MoreEvents extends StatelessWidget {
...
+    return InkWell(
+      onTap: onTap,
+      child: Container(
+        padding: const EdgeInsets.symmetric(
+          vertical: 3,
+          horizontal: 5,
+        ),
+        child: Text(
+          '$value ${onMoreEventsLabel ?? ''}',
+          style: textStyle ?? textTheme.bodyMedium!.copyWith(fontSize: 10),
```
Comment (raelsj) [inline lib/src/more_events.dart:32]: see above

**Review by raelsj — CHANGES_REQUESTED** [review]

**Review by raelsj — APPROVED** [review]

**Pull request merged**